WooCommerce/Classes/ViewRelated/Orders/Order Details/Product List Section/Fulfillment/FulfillViewController.swift

```diff
@@ -630,24 +630,25 @@ private extension FulfillViewController {
             return Section(title: title, secondaryTitle: nil, rows: [row])
         }()
 
-        let address: Section = {
+        let address: Section? = {
             var rows: [Row] = []
 
             if shippingLines.count > 0 {
                 rows.append(.shippingMethod)
             }
 
+            let orderContainsOnlyVirtualProducts = self.products?.filter { (product) -> Bool in
+                return self.order.items.first(where: { $0.productID == product.productID}) != nil
```

Comment:
nit: `self` in `self.order` and `self.products` in the previous line might not be necessary?